source/codegen/metadata/niswitch/functions.py

```diff
@@ -301,7 +314,6 @@
                 'type': 'ViSession'
             },
             {
-                'attribute': 'SCAN_DELAY',
```

Comment:
Note: We talked about this offline and this 'attribute' key/value isn't used in grpc-device. Doesn't affect generated output. Safe to remove.